frontend/src/vars.styl

```diff
@@ -21,8 +21,7 @@ $listen = #A66321
 $key-top = #222
 $key-bottom = #444
 
-$operation-status-colors = {
-  // Keys here are the backend representations of OperationStatus defined in backend/models
+$challenge-colors = {
   '0': #FFC940,
   '1': #3DCC91,
   '2': #FF7373,
```

Comment:
This makes sense for now, but we might want to change how this is represented. Also, we might not need all of the colors. The mapping was to help tie the status to the color, but since there's no status, it's probably better to call these something like $challenge-warning vs $challenge-colors[1]

Reply:
changed